admin/public/styles/react/react.less

```diff
@@ -37,3 +37,6 @@
 @import "@{reactSelectPath}/less/select.less";
 
 @import "../../../../fields/types/markdown/less/bootstrap-markdown.less";
+
+/* The path to keystone-tinymce is dynamically detected by KeystoneJS */
+@import "@{keystoneTinymcePath}/skin/skin.less";
```

Comment:
Was removed from https://github.com/keystonejs/keystone/pull/4897/files#diff-b854f0bd1f70678de625203f781c39d2L40

So I've added a similar path back in it's place